src/library/commands.ts

```diff
@@ -1,22 +1,15 @@
-import glob from 'glob';
-import config from '../config';
-import CommandLoader, { ICommandClasses } from './commandLoader';
+import { ICommandClasses } from './commandLoader';
 import Command from './iCommand';
 
-export const COMMANDS_PATH_GLOB = './src/commands/*.ts';
-
 // TODO: debateable whether we even need this wrapper class
 /**
  * @class Commands
  */
 export default class Commands {
-
   public readonly all: ICommandClasses;
-  private commandFiles: string[];
 
-  constructor() {
-    this.commandFiles = glob.sync(COMMANDS_PATH_GLOB);
-    this.all = CommandLoader.getCommandClasses(this.commandFiles);
+  constructor(commandClasses: ICommandClasses) {
+    this.all = commandClasses;
```

Comment:
This is so much cleaner, less cohesion, and testable with Dependency Injection™ 🎉